content/posts/2023-08-22-create-microshift-next-bundle.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "A Step-by-Step Guide to Creating the MicroShift next Bundle"
+date: 2023-08-22T12:30:10+05:30
+---
+
+CRC is a tool that allows you to quickly and easily spin up a local OpenShift cluster.
+It is a great way to learn about OpenShift/OKD/MicroShift and to test out new features.
```

Comment:
do we want to also mention kubernetes here?

Reply:
I think when we say Openshift it is by default a kubernetes distribution.